GitLab/custom-executor.md

```diff
@@ -29,6 +56,8 @@ To set up a GitLab Runner, you need to:
 
     **Note**: A template config file is available [here](template-config.md).  
     For information about the location of the file and its contents, see the GitLab Runner [configuration page][config-page] and the Custom executor [configuration page][custom-config-page].
+7. Verify [jq][jq] is installed. For more information, see [jq][jq] page.
+8. Verify that the private SSH key created [earlier](#set-up-an-orka-vm-base-image) that is used to connect to the ephemeral VM is present on the Runner machine and added to the ssh-agent.
```

Comment:
Verify that the private SSH key for connecting to the ephemeral agent is present on the wrapper machine and added to the ssh-agent. This key was created earlier during the Orka base image setup.